wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -46,7 +46,7 @@ public class MenuSelectionsController : ApiController
                 OrdererName = selection.OrdererName,
                 Description = selection.Description,
             };
-
+            selectionEntity.FoodOrder = order;
```

Comment:
I'll keep this in mind moving forward. Might be a good idea to protect `selection.FoodOrder` in that case.